_config.yml

```diff
@@ -19,14 +19,17 @@ languages:
   title: Woordelys
 - key: en
   name: English
+  title: Glossary
```

Comment:
Why two title entries per key for these 3 languages?

Reply:
Spanish and French have the same problem.

Reply:
That was a merge glitch - fixed.